order/src/main/resources/messages/messages.properties

```diff
@@ -0,0 +1,2 @@
+ORDER_NOT_FOUND=ORDER {} is not found
```

Comment:
I think we should not capitalize like this. ORDER_NOT_FOUND=Order {} is not found

Reply:
thanks